Tests/js/views/assets/ez-trashview-tests.js

```diff
@@ -269,6 +270,65 @@ YUI.add('ez-trashview-tests', function (Y) {
 
             this.view.fire('whatever:restoreTrashItemsAction');
         },
+
+        "Should fire `restoreItems` with destination when restoring a single item": function () {
```

Comment:
this test should be split in 3 tests:

* run the UDW
* allow to pick only container
* fire restoreItems with the expected parameters